client/state/preferences/actions.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Internal dependencies
+ */
+import wpcom from 'lib/wp';
+import {
+	PREFERENCES_SET,
+	PREFERENCES_RECEIVE,
+	PREFERENCES_REMOVE,
+	PREFERENCES_FETCH
+} from 'state/action-types';
+
+export const USER_SETTING_KEY = 'calypso_preferences';
+
+export function fetchPreferences() {
+	return ( dispatch ) => {
+		dispatch( { type: PREFERENCES_FETCH } );
+		return wpcom.me().settings().get().then( ( data ) => {
+			dispatch( {
+				type: PREFERENCES_RECEIVE,
+				data
+			} );
+		} );
+	};
+}
+
+const save = () => ( dispatch, getState ) => {
+	const settings = {};
+	settings[ USER_SETTING_KEY ] = getState().preferences.values;
+	wpcom.me().settings().update( JSON.stringify( settings ) ).then( ( data ) => {
+		dispatch( {
+			type: PREFERENCES_RECEIVE,
+			data
+		} );
+	} );
+};
+
+export const setPreference = ( key, value ) => ( dispatch, getState ) => {
+	dispatch( {
+		type: PREFERENCES_SET,
+		key,
+		value
+	} );
+	save()( dispatch, getState );
+};
+
+export const removePreference = ( key ) => ( dispatch, getState ) => {
```

Comment:
I don't think we use it.
That being said, there's a couple of components that use flux preferences.
After landing this PR, I plan on tackling them. Once we move all preferences to redux and still there will be no need for remove, then I'll get rid of it.